src/Microsoft.VisualBasic/tests/UnitTests/Microsoft/VisualBasic/Devices/ComputerTests.cs

```diff
@@ -3,9 +3,7 @@
 
 namespace Microsoft.VisualBasic.Devices.Tests;
 
-[Collection("Sequential")]
-[CollectionDefinition("Sequential", DisableParallelization = true)]
-[UISettings(MaxAttempts = 3)] // Try up to 3 times before failing.
+// This class does not access system singletons, it is safe to run in parallel.
```

Comment:
Move to the clipboard method

```suggestion
// This method does not actually access the clipboard and as such, it is safe to run in parallel.
```

Reply:
Sequential collection is not dedicated to the clipboard. It's used for mouse position as well.